Dockerfile

```diff
@@ -14,8 +14,10 @@ RUN apt-get update -qq && apt-get -y --no-install-recommends install \
 WORKDIR primace
 COPY . .
 
-RUN Rscript -e 'install.packages("renv", repos = "https://rstudio.r-universe.dev")' 
+RUN Rscript -e 'install.packages("renv", repos = "https://rstudio.r-universe.dev")'
 RUN Rscript -e 'library(renv)' 
+RUN Rscript -e 'install.packages("remotes")'
+RUN Rscript -e 'remotes::install_version("mlr3viz", version = "0.5.10")'
```

Comment:
Dockerfile try to restore `mlr3viz` to latest version after we manually installing it . so to prevent that it seems necessary to put `renv::settings$ignored.packages("mlr3viz")`  to prevent restoring it from version `0.5.10` to `0.6`

Reply:
ok, so i will put `renv::settings$ignored.packages("mlr3viz")` before `renv::restore()`

Reply:
ok i will give it a try. thanks for review